src/app/shared/player/player.component.html

```diff
@@ -68,6 +68,8 @@ <h2 [title]="subtitle">{{subtitle}}</h2>
       </play-progress>
       <span class="duration" [class.loaded]="duration | async ">{{duration | async | duration}}</span>
     </div>
+
+    <div class="overlayContent">{{overlayText}}</div>
```

Comment:
How did you get newlines to appear in here?  SafeHtml or something?

Reply:
I just checked this out and found that newlines are escaped, like you suspected. Chris suggested that we update this soon in a follow-up PR, once we start putting content in.